### ISSUE TITLE
Multiple Redactor controls do not work within a Vizy block

### ISSUE THREAD

**Issue by dyerc** [verbb/vizy#155]
**Description**
I added a second Redactor field to a Vizy block in order to provide an editable table. I found that although I could insert a table in the second Vizy block, once it was inserted none of the controls worked. This seems to affect all? formatting within the second Redactor field, bold, italic, lists etc.

**Steps to reproduce**
1. Create a Vizy block with multiple Redactor fields
2. Attempt to use any formatting controls eg. bold, italic, table etc.

**Additional info**
- Plugin version: 1.0.13, 1.0.15 and 2.0.1
- Craft version: 3.7.44 and 4.0.3
- Multi-site: No

**Additional context**
In the screenshot below I am unable to make the word "Test" in the bottom Redactor field bold.

<img width="734" alt="Screenshot 2022-06-15 at 12 36 24" src="https://user-images.githubusercontent.com/353185/173818209-f73c2b70-5869-4d93-879b-b4b741909671.png">

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.15",
  "...": "..."
}
```

Then run `composer update`.

**Comment by aodihis** [comment]
@engram-design, for the craft 4 plugin is there any early fix for this one?

**Comment by engram-design** [comment]
@aodihis Not yet sorry, but expect a release in the next few days.

**Comment by engram-design** [comment]
Fixed in [1.0.16](https://github.com/verbb/vizy/releases/tag/1.0.16) and [2.0.3](https://github.com/verbb/vizy/releases/tag/2.0.3)
